cmd/subnetcmd/upgradecmd/install.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/ava-labs/avalanche-cli/pkg/binutils"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/subnet/upgrades"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	ANRclient "github.com/ava-labs/avalanche-network-runner/client"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+const (
+	timestampFormat  = "20060102150405"
+	tmpSnapshotInfix = "-tmp-"
+)
+
+// avalanche subnet upgrade generate
+func newUpgradeInstallCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install [subnetName]",
+		Short: "Installs upgrade bytes onto subnet nodes",
+		Long:  `Installs upgrade bytes onto VMs.`, // TODO fix this wording
```

Comment:
Your long is shorter than your short lol

Reply:
Apply generated upgrade bytes to running subnet nodes to trigger a network upgrade.

Reply:
Indeed, hence the TODO. Thanks for the suggestion.